redirect/src/main/java/com/adyen/checkout/redirect/RedirectComponent.java

```diff
@@ -60,22 +54,12 @@ public static String getReturnUrl(@NonNull Context context) {
     /**
      * Make a redirect from the provided Activity to the target of the Redirect object.
      *
-     * @param activity The Activity starting the redirect.
+     * @param activity       The Activity starting the redirect.
      * @param redirectAction The object from the server response defining where to redirect to.
      */
     public static void makeRedirect(@NonNull Activity activity, @NonNull RedirectAction redirectAction) throws ComponentException {
-        Logger.d(TAG, "makeRedirect - " + redirectAction.getUrl());
-        if (!TextUtils.isEmpty(redirectAction.getUrl())) {
-            final Uri redirectUri = Uri.parse(redirectAction.getUrl());
-            final Intent redirectIntent = RedirectUtil.createRedirectIntent(activity, redirectUri);
-            try {
-                activity.startActivity(redirectIntent);
-            } catch (ActivityNotFoundException e) {
-                throw new ComponentException("Redirect to app failed.", e);
-            }
-        } else {
-            throw new ComponentException("Redirect URL is empty.");
-        }
+        final RedirectDelegate delegate = new RedirectDelegate();
```

Comment:
@caiofaustino I tried to move this field to the component itself and inject it in the provider (just like `Adyen3DS2Component.redirectDelegate`) but this method is static. I didn't change it to non static because I supposed we don't need to initialize the component to be able to make a redirect, am I right?